src/bundle/ui-dev/src/modules/sub-items/sub.items.module.js

```diff
@@ -240,10 +252,17 @@ export default class SubItemsModule extends Component {
     loadPage(pageIndex) {
         const { limit: itemsPerPage, parentLocationId: locationId, loadLocation, restInfo } = this.props;
         const { sortClause, sortOrder } = this.state;
-        const page = this.state.pages.find(({ number }) => number === pageIndex + 1);
-        const cursor = page ? page.cursor : null;
+        const cursor = this.calculateCursor(pageIndex);
         const queryConfig = { locationId, limit: itemsPerPage, sortClause, sortOrder, cursor };
 
+        this.setState(() => ({
+            queryParams: {
+                sortClause,
+                sortOrder,
+                cursor,
+            },
+        }));
```

Comment:
nitpick: this can be simplified
```suggestion
        this.setState({
            queryParams: {
                sortClause,
                sortOrder,
                cursor,
            },
        });
```